Jenkinsfile

```diff
@@ -27,15 +27,31 @@ pipeline {
 
         stage('Build') {
             steps {
+                sh "docker login quay.io -u ${QUAY_IO_CREDS_USR} -p ${QUAY_IO_CREDS_PSW}"
                 sh "make build-image build-minimal-assisted-iso-generator-image"
-                sh "make jenkins-deploy-for-subsystem"
-                sh "kubectl get pods -A"
+                sh "make update"
```

Comment:
I'm not sure that we should invoke 'make update' here.
Lets discuss this f2f

Reply:
> I'm not sure that we should invoke 'make update' here.
> Lets discuss this f2f

I added `quay.expires-after` label